archetypes/catalog/cli-metadata.properties.mustache

```diff
@@ -13,8 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-build-tools.version={{buildtoolsVersion}}
+build-tools.version=2.0.2
```

Comment:
Seems to be the wrong version here, shouldn't this be the placeholder?

Reply:
Romain said this was intentional as it was "the version of the helidon-maven-plugin that contains the goal “dev” before we split it into helidon-cli-maven-plugin". And that this is used only for testing. We might want to follow-up on this, but I don't think it should block the PR.

Reply:
This property is correct. It corresponds to the older version of `helidon-maven-plugin` that contained the goal `dev`, it is used for backward compatibility. Also note that this file is only used for testing.